scripts/e2e-tests.sh

```diff
@@ -197,23 +232,23 @@ function create_test_cluster() {
   # Don't fail test for kubetest, as it might incorrectly report test failure
   # if teardown fails (for details, see success() below)
   set +o errexit
-  run_go_tool k8s.io/test-infra/kubetest \
+  { run_go_tool k8s.io/test-infra/kubetest \
     kubetest "${CLUSTER_CREATION_ARGS[@]}" \
     --up \
     --extract "${E2E_CLUSTER_VERSION}" \
     --gcp-node-image "${SERVING_GKE_IMAGE}" \
     --test-cmd "${E2E_SCRIPT}" \
     --test-cmd-args "${test_cmd_args}" \
     ${extra_flags[@]} \
-    ${EXTRA_KUBETEST_FLAGS[@]}
+    ${EXTRA_KUBETEST_FLAGS[@]}; } 2>&1 | tee $CLUSTER_CREATION_LOG
```

Comment:
I wonder if kubetest returns a different error code in case cluster creation fails, so you don't have to grep the log?

Reply:
No. Proof:

https://github.com/kubernetes/test-infra/blob/2447e896ea6091ff5c6c500f20c754bf05d7f8d4/kubetest/main.go#L308

https://golang.org/pkg/log/#Logger.Fatalf always exits with code 1.